sklearn/experimental/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+The :mod:`sklearn.experimental` module hosts experimental functionality for
+which the API is not yet guaranteed to be stable.
+"""
+
+from ._column_transformer import *
```

Comment:
I would prefer to avoid an import * and import only the symbols that we are using. Also, I humbly think that we should have an "__all__".